expo/tests/contexts/act_techinfo_context.py

```diff
@@ -0,0 +1,46 @@
+from tests.factories.gbe_factories import (
+    ActFactory,
+    ConferenceFactory,
+    CueInfoFactory,
+    PersonaFactory,
+    RoomFactory,
+    ShowFactory,
+)
+from tests.factories.scheduler_factories import (
+    ActResourceFactory,
+    LocationFactory,
+    ResourceAllocationFactory,
+    SchedEventFactory,
+)
+
+class ActTechInfoContext():
+    def __init__(self,
+                 performer=None,
+                 act=None,
+                 show=None,
+                 conference=None,
+                 room_name=None,
+                 cue_count=1):
+        self.conference = conference or ConferenceFactory()
+        self.performer = performer or PersonaFactory()
+        self.act = act or ActFactory(performer=self.performer,
+                                     conference=self.conference,
+                                     accepted=3)
+        self.tech = self.act.tech
+        self.audio = self.tech.audio
+        self.lighting = self.tech.lighting
+        self.stage = self.tech.stage
+        for i in range(cue_count):
+            CueInfoFactory.create(techinfo=self.tech)
```

Comment:
For realism on this, I might suggest that cue_sequence be set to "i" - in the system they are sequentially numbered starting at 1 w/in each act...